src/InsertionsClient.Core/Api/Providers/InsertionApi.cs

```diff
@@ -229,6 +231,13 @@ internal bool TryExtractManifestAssets(string manifestFile, ICollection<Asset> a
 
                 foreach (Build build in buildManifest.Builds)
                 {
+                    if(buildFilter != null && !buildFilter(build))
+                    {
+                        Trace.WriteLine($"The build with {nameof(build.BuildNumber)} {build.BuildNumber} was ignored by the filter. " +
+                            "Assets from this build will not be used in the insertion process.");
+                        continue;
+                    }
+
```

Comment:
Should this message be adapted in case manifest file contains the builds but all of them were filtered out?
```
if (initialCollectionSize == assets.Count)
{
     details = $"No assets were found in the manifest file.";
}
```

Reply:
Thanks, I updated accordingly.